book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
+    library1 =  Library.objects.create(
+        id=1,
+        name='Library1',
+        street='1234 Library1 St.',
+        city='Library1 City',
+        state='CA',
+        zip='12345',
+        lat=123.456,
```

Comment:
Are these actual valid `lat` and `lon`? At some point you'll probably want validation to make sure that's true so I'd make sure to use examples that fit that definition. These might be! Just wanted to call that out.

Reply:
They aren't really - we'll update with actual lat and lon coordinates

Reply:
They are not valid, we were just cruising through the tests with some made up data but you make a great point.  I was curious if we use Faker if it can generate valid latitude and longitude values.

Reply:
Nvm, I just checked and Faker can indeed generate them.  Learning something new about Faker :)

Reply:
Yeah, it's pretty rare to run into a situation where you need something that Faker can't provide for you, it's a great package!